src/cwe_checker_lib/src/intermediate_representation/sub.rs

```diff
@@ -151,6 +152,16 @@ impl ExternSymbol {
     }
 }
 
+impl fmt::Display for ExternSymbol {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "[{}] n:{}", self.tid, self.name)?;
+        for addr in self.addresses.iter() {
+            write!(f, " a:{}", addr)?;
```

Comment:
Again, I do not like the 1-letter-abbreviations in the output when they are not necessary.